src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs

```diff
@@ -35,23 +35,4 @@ public EnumerableDebugView(IEnumerable<KeyValuePair<TKey, TValue>> enumerable)
         [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
         public KeyValuePair<TKey, TValue>[] Items { get { return _enumerable.ToArray(); } }
     }
-
-    /// <summary>Debugger type proxy for an enumerable of T.</summary>
-    internal sealed class EnumerableDebugView<T>
```

Comment:
Good catch. How is this still compiling?
```c#
    [DebuggerTypeProxy(typeof(EnumerableDebugView<,>))]
    internal sealed class QueuedMap<TKey, TValue>
```

@erozenfeld what am I missing here?